src/view.c

```diff
@@ -611,6 +612,41 @@ view_get_edge_snap_box(struct view *view, struct output *output,
 	return dst;
 }
 
+/* Move and resize view based on percent output local values in rel_box */
+void
+view_rel_move_resize_to(struct view *view, struct wlr_box *rel_box)
```

Comment:
`rel` and `rel_box` misled me here because I was thinking relative vs absolute. Other openbox actions using `relative` in that latter sense, but it'd be good to think of another term.

Reply:
Hm.. right. It \*is\* relative but relative to the primary output of the view.